content/sensu-go/5.16/api/role-bindings.md

```diff
@@ -122,7 +140,8 @@ The `/rolebindings/:rolebinding` API endpoint provides HTTP GET access to [role
 In the following example, querying the `/rolebindings/:rolebinding` API endpoint returns a JSON map that contains the requested [`:rolebinding` definition][1] (in this example, for the `:rolebinding` named `readers-group-binding`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings/readers-group-binding -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings/readers-group-binding
```

Comment:
requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency